src/coinbase/wallet.ts

```diff
@@ -130,31 +136,67 @@ export class Wallet {
   }
 
   /**
-   * Imports a Wallet for the given Wallet data.
+   * Loads an existing CDP Wallet using a wallet data object or mnemonic seed phrase.
```

Comment:
Can we document that it creates a new wallet for mnemonic seed phrases?

Or how can we best clarify that this leaves the existing behavior + adds new behavior, and lays the groundwork for the breaking change in the next release?

Not a blocker per se, but we should aim to make this clear in our public docs + here when we make the cutover